src/db/cassandra/cluster_services.py

```diff
@@ -0,0 +1,48 @@
+import re
+
+class ClusterServices:
+  """The ClusterServices class provides methods used to create the keyspaces and tables 
+     for the Meowl application
+  
+  Attributes:
+      cluster: The Cassandra cluster for this instance
+      session: The current session
+  """
+  def __init__(self, cluster):
+    self.cluster = cluster
+    self.session = self.cluster.connect()
+
+
+  def get_session(self):
```

Comment:
Why do you need a getter for a public variable?

Reply:
yes